scripts/haas_network.service

```diff
@@ -0,0 +1,18 @@
+[Unit]
+Description=HaaS Network Server 
+After=network.target
```

Comment:
I like the `After=` option, since it doesn't introduce a hard requirement that the DB is on the same machine